bitnami/kubeapps/Chart.yaml

```diff
@@ -25,4 +25,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 6.0.1
+version: 6.1.1
```

Comment:
This is tricky... we released the 6.1.0, but it never got synced due to the permissions issue. Later on, we bumped it to 6.1.1 (which is the current version in Kubeapps master).
In order to avoid semver problems, perhaps we better talk about what to do here (fix the permissions and re-run the sync from our side, split this PR into two, etc.)

Reply:
Fair enough!